src/ngraph/op/util/arithmetic_reduction.hpp

```diff
@@ -29,6 +29,16 @@ namespace ngraph
             class ArithmeticReduction : public Op
             {
             public:
```

Comment:
Probably want to change this one to `protected`, like you did in `BinaryElementwiseArithmetic`.

Reply:
done